tests/hpc/slurm_master.pm

```diff
@@ -425,15 +437,22 @@ sub run ($self) {
     my $nodes = get_required_var('CLUSTER_NODES');
     my $slurm_conf = get_required_var('SLURM_CONF');
     my $version = get_required_var('VERSION');
-
+    if ($slurm_pkg =~ /slurm/) {
+        my $query_output = script_output(qq{zypper if slurm | grep -i version | awk '{print \$3}'});
```

Comment:
I will recommend as usual what i recommend when I see awk and grep in pipe : https://blog.jpalardy.com/posts/skip-grep-use-awk/

Reply:
Done, ready for review